draft-terbu-sd-jwt-vc.md

```diff
@@ -71,88 +139,117 @@ Credentials are cryptographically signed statements about a Subject, typically t
-aims to express the basic
-Verifiable Credential data model purely through JWT Claim Sets, using registered
-claims while allowing Issuers to use additional registered claims, as well as
-new or existing public claims, to make statements about the Subject of the
+Verifiable Credentials in a way that is easy to understand and process
+as it builds upon established web primitives. While JWT-based
+credentials enable selective disclosure, i.e., the ability for a Holder
+to disclose only a subset of the contained claims, in an Identity
+Provider ecosystem by issuing new JWTs to the Verifier for every
+presentation, this approach does not work in the three-party-model.
+
+Selective Disclosure JWT (SD-JWT)
+[@!I-D.ietf-oauth-selective-disclosure-jwt] is a specification that
+introduces conventions to support selective disclosure for JWTs: For an
+SD-JWT document, a Holder can decide which claims to release (within
+bounds defined by the Issuer). This format is therefore perfectly suited
+for Verifiable Credentials.
+
+SD-JWT itself does not define the claims that must be used within the
+payload or their semantics. This specification therefore defines how
+Verifiable Credentials can be expressed using SD-JWT.
+
+JWTs (and SD-JWTs) can contain claims that are registered in "JSON Web
+Token Claims" registry as defined in [@!RFC7519], as well as public and
+private claims. Private claims are not relevant for this specification
+due to the openness of the three-party-model. Since SD-JWTs are based on
+JWTs, this specification aims to express the basic Verifiable Credential
+data model purely through JWT Claim Sets, using registered claims while
+allowing Issuers to use additional registered claims, as well as new or
+existing public claims, to make statements about the Subject of the
 Verifiable Credential.
 
-## Requirements Notation and Conventions
+# Native JWT Representation of Verifiable Credentials
 
-The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
-"SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this
-document are to be interpreted as described in RFC 2119 [@!RFC2119].
+## Overview
 
-## Terms and Definitions
+This section provides guidance on how to use JSON [@!RFC8259] claimsets
+with JWT [@!RFC7519] registered claims to construct a JWT that can be
+mapped to a verifiable credential. This section also describes how to
+use content types and token types to distinguish different
+representations of verifiable credentials.
 
-This specification uses the terms "Holder", "Issuer", "Verifier", defined by
-[@!I-D.ietf-oauth-selective-disclosure-jwt].
+This representation relies on claims registered in the [IANA JSON Web
+Token Claims
+Registry](https://www.iana.org/assignments/jwt/jwt.xhtml#claims)
+whenever possible.
 
-Verifiable Credential (VC):
-:  An Issuer-signed assertion with claims about a Subject.
+Implementers using this representation SHOULD NOT use `vc+ld+json` as an
```

Comment:
sounds like a self fulfilling prophecy - why is that needed? I would rather state what the expected `typ` value is.